### DISCUSSION TITLE
Allow a good vertical alignment of 5 lines of text

### DISCUSSION THREAD

**Discussion by danyallard** [ni/labview-icon-editor#186] [General]
As a LabVIEW user I want that the icon editor allow a good vertical alignment of 5 lines of text by removing the black line at the bottom of the header. As shown in the next snapshot 

![image](https://github.com/ni/labview-icon-editor/assets/3820056/3acfc73c-5848-415e-90a7-0aea3b62da80)

**Comment by j-medland** [comment]
Hi @svelderrainruiz this had the tag more-clarification-needed added but no additional details on what that clarification should be

**Comment by svelderrainruiz** [comment]
Will move to a discussion after talking about it on todays steerco meeting

**Comment by svelderrainruiz** [comment]
This looks like a good feature to have, but for it to be automatically disabled initially. Having this feature be enabled via a checkbox  as an option could be an alternative.
